rootfs/etc/profile.d/_01-launch-warning.sh

```
@@ -0,0 +1,63 @@
+	for i in $(seq 1 $xb); do
+		bx+=$'\b'
+		fx+=" "
+	done
+	m+=$bx
+	m+=$fx
+	m+=$bx
+
+	# Cover the eval command itself. Tough to compute the exact number of backspaces needed,
+	# due to escapes and non-printing characters. So we just add a few extra, because
+	# extra backspaces are ignored.
+	feval=$(printf "%*s" 18)
```

Shellcheck (0.10.0)

[warning] This format string has 2 variables, but is passed 1 argument.

(SC2183)